plugins/woocommerce/package.json

```diff
@@ -577,9 +585,14 @@
 						"--shard=10/10"
 					],
 					"changes": [
-						"src/Blocks/**",
-						"templates/**",
-						"patterns/**"
+						"patterns/**/*.php",
```

Comment:
Changes (Blocks e2e tests):
- narrow down patterns a bit more specifically
- trigger only on changes in two specific packages (other packages should cover each other with plain JS tests and running the job on trunk merge)